### ISSUE TITLE
declaration, not statement

### ISSUE THREAD

**Issue by Kaelum** [dotnet/docs#34354]
The following text is incorrect:

> The newer using statement syntax translates to similar code. The try block opens where the variable is declared. The finally block is added at the close of the enclosing block, typically at the end of a method.

It should read:

> The newer using declaration syntax translates to similar code. The try block opens where the variable is declared. The finally block is added at the close of the enclosing block, typically at the end of a method.

Further down the page, you might also want to add the following format, which is much nicer to read:

```
using (StringReader left = new StringReader(numbers)
using (StringReader right = new StringReader(letters))
{
    string? item;
    do
    {
        item = left.ReadLine();
        Console.Write(item);
        Console.Write("    ");
        item = right.ReadLine();
        Console.WriteLine(item);
    } while (item != null);
}
```

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 042902cc-1598-5df1-34cc-caa0fad63e02
* Version Independent ID: e186d0c4-523e-38f7-2cde-d00109ea784b
* Content: [using statement - C# Reference](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement?f1url=%3FappId%3DDev16IDEF1%26l%3DEN-US%26k%3Dk(using-statement_CSharpKeyword)%3Bk(SolutionItemsProject)%3Bk(DevLang-csharp)%26rd%3Dtrue)
* Content Source: [docs/csharp/language-reference/keywords/using-statement.md](https://github.com/dotnet/docs/blob/main/docs/csharp/language-reference/keywords/using-statement.md)
* Product: **dotnet-csharp**
* Technology: **csharp-language-reference**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

**Comment by BillWagner** [comment]
Thanks @Kaelum 

I like the additional example.

As for the distinction between "using declaration" and "using statement", both are correct. A variable declaration *is* a statement. We should consider updated wording base on dotnet/csharpstandard#672. The standards committee will likely add the "using" and "async" declarations in the area of a variable declaration statement, That would make this change more consistent.

**Comment by Kaelum** [comment]
@BillWagner however, in the context of what is being stated, only declaration is correct.  It's the differentiation between a statement block, and the block containing the declaration.  That specific paragraph is only correct for the latter (declaration) and not the former (statement).

**Comment by BillWagner** [comment]
Fixed in #34516
